addons/event/i18n/event.pot

```diff
@@ -3356,15 +3364,6 @@ msgstr ""
 msgid "What's new?"
 msgstr ""
 
-#. module: event
-#. openerp-web
-#: code:addons/event/static/src/js/tours/event_tour.js:0
-#, python-format
-msgid ""
-"When will your event take place? <b>Select</b> the start and end dates "
```

Comment:
We do not remove strings in stable, that way people that do not update their source code (or don't do F5 for js files) are not impacted. It will be cleaned in master when pot files are synchronized again :) . So keep only new strings in stable, do not remove other one (except if they are linked to a field as there are some constraints, which is not the case here).